rpm/wasmedge.spec.in

```diff
@@ -45,15 +36,26 @@ cmake --build .
 %install
 cd build
 DESTDIR=%{buildroot} cmake --build . --target install
+mv %{buildroot}%{_libdir}/lib%{name}_c.so %{buildroot}%{_libdir}/lib%{name}_c.so.%{gittag}
+mv %{buildroot}%{_libdir}/%{name}/lib%{name}PluginWasmEdgeProcess.so %{buildroot}%{_libdir}/%{name}/lib%{name}PluginWasmEdgeProcess.so.%{gittag}
+ln -s lib%{name}_c.so.%{gittag} %{buildroot}%{_libdir}/lib%{name}_c.so
+ln -s lib%{name}PluginWasmEdgeProcess.so.%{gittag} %{buildroot}%{_libdir}/%{name}/lib%{name}PluginWasmEdgeProcess.so
 
 %files
+%license LICENSE LICENSE.spdx
```

Comment:
If LICENSE.spdx is applicable to the entire package or any of the files installed by the package, the `License` line at the top of the spec file should change to: `License: ASL 2.0 and CC0`

Reply:
If the spdx file doesn't apply to any of the files in this package, then it's ok to remove it.

Reply:
Yes, `LICENSE.spdx` applies to some files in this package. I think we should change the `License:` line to `License: ASL 2.0 and CC0`. Appreciate your help!

Reply:
Done. Thanks